Exiled.API/Features/Player.cs

```diff
@@ -783,6 +783,20 @@ public bool IsIntercomMuted
         /// </summary>
         public bool IsSpeaking => VoiceModule != null && VoiceModule.IsSpeaking;
 
+        /// <summary>
+        /// Gets the loudness of a player when speaking.
+        /// </summary>
+        public float Loudness
+        {
+            get
+            {
+                if (!IsSpeaking || VoiceModule == null || VoiceModule is not StandardVoiceModule standardModule)
```

Comment:
```suggestion
                if (!IsSpeaking || VoiceModule is not StandardVoiceModule standardModule)
```
Is not needed since IsSpeaking already contain this checkc